docs/spatial-vector.rst

```diff
@@ -1,67 +0,0 @@
-Earthpy Spatial Vector Data
```

Comment:
Sure - I put these in `docs/ignoreme` and modified `conf.py` to ignore those files (otherwise the build fails if it sees rst files that are not indexed).